extensions/quarto-stock-report-python/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": 1,
+  "locale": "en_US.UTF-8",
+  "metadata": {
+    "appmode": "quarto-static"
+  },
+  "extension": {
+    "name": "quarto-stock-report-python",
+    "title": "Quarto Stock Report using Python",
```

Comment:
Thinking about the titles, not sure if any change is necessary:

The IDE jumps to using `quarto-stock-report-python` which isn't particularly helpful. Publisher uses "Stock Report" to match the Quarto project title. The title you have may be the most helpful.

Reply:
Makes perfect sense. I think the name of the content changes based on how you deploy though which is what I was trying to note above.